HandsOff/UI/VideoPlayerView.swift

```diff
@@ -0,0 +1,38 @@
+import AVKit
+import SwiftUI
+
+struct VideoPlayerView: View {
+    let videoName: String
+
+    var body: some View {
+        GeometryReader { geometry in
+            if let url = Bundle.main.url(forResource: videoName, withExtension: "mp4") {
+                let player = AVPlayer(url: url)
+                VideoPlayer(player: player)
+                    .aspectRatio(contentMode: .fill)
+                    .frame(width: geometry.size.width, height: geometry.size.height)
+                    .edgesIgnoringSafeArea(.all)
+                    .onAppear {
+                        player.play()
+                    }
+
+            } else {
+                // Fallback if video is not found
```

Comment:
this might not be required since we know video will exist for sure, but didn't want to ! anything 😂